Automattic/wp-calypso
# Update the list of popular verticals for Blog and Professional segments

#### Changes proposed in this Pull Request

This Pull Request updates the list of popular verticals for Blog and Professional segments with entries from the new verticals list.

#### Testing instructions

1. Choose Blog/Professional in the site type step
2. Confirm the list of suggestions is specific to that segment
